src/components/ec-smart-table/__snapshots__/ec-smart-table.spec.js.snap

```diff
@@ -461,6 +461,22 @@ exports[`EcSmartTable filtering should handle changes in filters and reload the
     Test 2 (N/A)
   </div>
   
+  <!-- TODO ONL-4893
```

Comment:
There's a this part in the jest.config to configure the compiler:
```
'vue-jest': {
      compilerOptions: {
        whitespace: 'condense',
        isCustomElement: tag => customElements.has(tag),
        comments: false,
      },
    },
```
It disables the comments, so they are not rendered into the HTML. It seems like this option is now not working. We need to check it and revert all these snapshots.

Reply:
Yes, It is solved by changing the transformer you've mentioned upper. The minor version 1 -> 3 in the alpha build of @vue/vue3-jest makes sense, I guess.